.github/workflows/check_imports.yml

```diff
@@ -0,0 +1,25 @@
+name: check_imports
+on: [pull_request]
```

Comment:
We should run this on push too like the other tests

Reply:
Definitely, let me add that right away